### DISCUSSION TITLE
yield reserved word

### DISCUSSION THREAD

**Discussion by 0xMySt1c** [graphprotocol/graph-tooling#1585] [General]
because yield is a reserved word in es6 strict mode it causes build to fail when the topic of an event is yield

**Comment by saihaj** [comment]
I suppose it is code generation causing this issue? I wonder what TypeChain does 🤔

**Reply by 0xMySt1c** [reply]
actually codegen runs fine.
its happening when the wasm bindings are created in build

**Comment by YaroShkvorets** [comment]
Tracking here: https://github.com/graphprotocol/graph-tooling/issues/1896
